packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -37,7 +37,16 @@ const trace = makeTracer(' VM ');
 // some number of outstanding loans, each called a Vault, for which the
 // collateral is provided in exchange for borrowed RUN.
 
-/** @type {MakeVaultManager} */
```

Comment:
`makeVaultManager` is being defined right here. No need to also name a type for it.